dbt/adapters/databricks/python_models/python_submissions.py

```diff
@@ -70,6 +71,8 @@ def __init__(
 
     @override
     def submit(self, compiled_code: str) -> None:
+        logger.debug("Submitting Python model using the Command API.")
```

Comment:
Adding debug statement so that we can quickly determine submission method from logs.